### ISSUE TITLE
Cosmos client Fetch error

### ISSUE THREAD

**Issue by thetre97** [cfworker/cfworker#75]
When I try to run a command with the Cosmos client, it always throws this error: `Illegal invocation`.
I found that if I changed L#339 in `client.ts` to `await fetch(request)` instead of `await this.systemFetch(request)`, the request then worked.

I am using the worker `webpack` template, Wrangler version `1.11.0`, and `@cfworker/cosmos` version `1.4.1`.

**Comment by jdanyow** [comment]
I think this line needs to change:
https://github.com/cfworker/cfworker/blob/920318f7a22c6f13b138d2092a2f52938ca8f31b/packages/cosmos/src/client.ts#L81

```diff
- this.systemFetch = config.fetch || fetch;
+ this.systemFetch = config.fetch || fetch.bind(self);
```

**Comment by jdanyow** [comment]
this is fixed in 1.6.0- thanks for the report
